scripts/run_mypy_checks.py

```diff
@@ -751,6 +750,12 @@
     help='If true, skips installing dependencies. The default value is false.',
     action='store_true')
 
+_PARSER.add_argument(
+    '--ci-check',
+    help='If true, installs mypy and its requirements globally.'
+    ' The default value is false.',
```

Comment:
This help message is confusing to me. It sounds like the user is supposed to pass `--ci-check=true`, but they're actually just supposed to pass `--ci-check`. This message might be clearer:

```suggestion
    help='Install mypy and its requirements globally. By default, they are installed to ' + MYPY_TOOLS_DIR,
```

(I see that you were following the pattern set by the argument defined above, but I think that's confusing too. No need to fix it though)

Reply:
Done